add bndtools workspace template fragment (#4685)
eclipse-tycho/tycho: setup/bnd-templates/pomless/.mvn/README.md

# Tycho Workspace template fragment for bndtools for pomless Maven build

> The is a [bndtools template fragment](https://bnd.bndtools.org/chapters/620-template-fragments.html) which you can use in a new or existing bndtools workspace, 

Reviewer: ```suggestion
This is a [bndtools template fragment](https://bnd.bndtools.org/chapters/620-template-fragments.html) which you can use in a new or existing bndtools workspace, 
```

to add a Maven build based on Eclipse Tycho for your bnd workspace.

> All it does is adding a `.mvn` folder to your bnd workspace, contain a `extensions.xml` and a `maven.config`.

Reviewer: ```suggestion
It creates a `.mvn` folder in your bnd workspace root, contain a `extensions.xml` and a `maven.config`.
```
